docs/src/content/docs/ru/challenges/angular/5-crud.md

```diff
@@ -0,0 +1,50 @@
+---
+title: 🟢 Crud приложение
+description: Задача 5 посвящена рефакторингу crud-приложения
+author: thomas-laforge
+challengeNumber: 5
+command: angular-crud
+sidebar:
+  order: 2
```

Comment:
you need to add your github handle into the contributors section now.

Reply:
added